tests/Domain/Fixture/FixtureCreatorTest.php

```diff
@@ -72,7 +75,11 @@ private function createFixtureCreator(): FixtureCreator {
     $output = $this->output->reveal();
     $subextension_manager = $this->subextensionManager->reveal();
     $version_finder = $this->versionFinder->reveal();
-    return new FixtureCreator($cloud_hooks_installer, $codebase_creator, $composer_facade, $composer_json_helper, $drupal_settings_helper, $fixture, $fixture_inspector, $git, $site_installer, $output, $process_runner, $package_manager, $subextension_manager, $version_finder);
+    $customizer = $this->customizer->reveal();
+    return new FixtureCreator($cloud_hooks_installer, $codebase_creator, $composer_facade,
+      $composer_json_helper, $drupal_settings_helper, $fixture, $fixture_inspector,
+      $git, $site_installer, $output, $process_runner, $package_manager, $subextension_manager,
+      $version_finder, $customizer);
   }
```

Comment:
More multi-line function parameters formatting:

```suggestion
    return new FixtureCreator(
        $cloud_hooks_installer,
        $codebase_creator,
        $composer_facade,
        $composer_json_helper,
        $drupal_settings_helper,
        $fixture,
        $fixture_inspector,
        $git,
        $site_installer,
        $output,
        $process_runner,
        $package_manager,
        $subextension_manager,
        $version_finder,
        $customizer
    );
```